nuclei/CVE-2023-7102.yaml

```diff
@@ -0,0 +1,74 @@
+id: CVE-2023-7102
+
+info:
+  name: Barracuda ESG Appliance Remote Code Execution
+  author: Mohamed Benchikh
+  severity: critical
+  description: |
+    Barracuda Email Security Gateway (ESG) appliances are vulnerable to a remote code execution through a specially crafted Microsoft Excel attachement.
+    In some cases it's not possible to extract the version of the Barracuda ESG appliance, therefore this template might miss some vulnerable instances.
+  reference:
+    - https://nvd.nist.gov/vuln/detail/CVE-2023-7102
+    - https://thehackernews.com/2023/12/chinese-hackers-exploited-new-zero-day.html
+  classification:
+    cvss-metrics: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
+    cvss-score: 9.8
+    cve-id: CVE-2023-7102
+    cwe-id: CWE-94,CWE-1104
+  metadata:
+    verified: true
+    max-request: 2
+    vendor: barracuda
+    product: esg
+    shodan-query: http.favicon.hash:1436966696
+    fofa-query: app="Barracuda-Spam-Firewall"
+  tags: cve-2023-7102,barracuda,rce,attachment
+
+flow: http(1) || http(2)
+
+http:
+  - method: GET
+    path:
+      - "{{BaseURL}}/cgi-mod/view_help.cgi"
+
+    matchers-condition: and
+    matchers:
+      - type: regex
+        part: body
+        regex:
+          - "/css/help.css\\?v=([5-8]\\.\\d{1,2}\\.\\d{1,2}.\\d{1,3}|9\\.[0-2]\\.\\d{1,3}|9\\.2\\.0{0,2}1)"
+
+      - type: status
+        status:
+          - 200
+
+    extractors:
+      - type: regex
+        part: body
+        group: 1
+        regex:
+          - "/css/help.css\\?v=([5-8]\\.\\d{1,2}\\.\\d{1,2}.\\d{1,3}|9\\.[0-2]\\.\\[0-1]$)"
```

Comment:
what does this part suppose to match please ?  : \\[0-1]$)

Reply:
The escape character was a typo